### PR TITLE
feat: dropdown multiselect component

### PR DESCRIPTION
**Pull request by hadalin**
Screenshot 2020-01-09 at 15 06 05

### CODE REVIEW

**Comment by hadalin** [comment]
There seems to be some strange problem with styling (maybe there's a bug in Storybook). The new DropdownMultiselect component that I created has some custom SCSS properties (to hide tags for example).

It works ok if you select it in the Storybook, but if you then select Multiselect component and DropdownMultiselect again, the latter inherits styling from Multiselect.

storybook

**Comment by leferreyra** [comment]
@hadalin code is looking good!

It seems that the checks don't work when you PR from a forked repo. Can you push your branch to the `reciprocity/zenlib` repo directly so all the checks will run? No need to do forks here like in zengrc.

File: packages/dropdown-multiselect/DropdownMultiselect.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    :close-on-select="false"
+    :clear-on-select="false"
+    :preserve-search="true"
+    :placeholder="placeholder"
+    :label="label"
+    :track-by="trackBy"
+    v-on="$listeners"
+  >
+    <template slot="selection" slot-scope="{ values, isOpen }">
+      <span v-if="values.length &amp;&amp; !isOpen" class="multiselect__single">
```
Comment (pr1moz) [inline packages/dropdown-multiselect/DropdownMultiselect.vue]:
```suggestion
      <span v-if="values.length && !isOpen" class="multiselect__single">
```

**Comment by hadalin** [comment]
>Can you also please make sure it looks mostly like the other Multiselect? In terms of font size, component height... We want all the form elements to be interchangeable (styling wise)

@pr1moz the styling defined in abstract [looks different](https://app.goabstract.com/projects/9064f6f0-cbc2-11e8-8c39-a92298c1f63e/branches/5b19bb87-e2a7-4d8c-9c39-4ea64733d260/collections/1bfb29ec-0913-4cb9-8b52-fee2d2781cd6?collectionLayerId=019d3c4c-b520-4b61-8fdd-972fc0e061d0). What is the source of truth, current implementation or what is there in the abstract?

**Comment by pr1moz** [comment]
@hadalin we discussed this with @mplesec on Friday - I think it's better to implement the current look for now and change it as part of the redesign effort. We want to avoid inconsistencies until we have dedicated time from product to systematically change the look of related components (forms in this case).

This is something that needs to be discussed with Klara today as well.

**Pull request closed without merging**